extensions/authentication-jwt/package.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "@loopback/extension-authentication-jwt",
+  "name": "@loopback/authentication-jwt",
```

Comment:
So the existing `@loopback/extension-authentication-jwt` is no longer going to be updated?

What about other packages with the `extension-` prefix (e.g. health, logging)?

Reply:
Other authentication related extensions only use `authentication-` prefix. That's why I rename this package to be consistent.

We'll continue to maintain other `extension-` packages.